xbmc/utils/StringUtils.h

```diff
@@ -104,9 +104,9 @@ class StringUtils
   static void ToLower(std::wstring &str);
   static void ToCapitalize(std::string &str);
   static void ToCapitalize(std::wstring &str);
-  static bool EqualsNoCase(const std::string &str1, const std::string &str2);
-  static bool EqualsNoCase(const std::string &str1, const char *s2);
-  static bool EqualsNoCase(const char *s1, const char *s2);
+  static bool EqualsNoCase(const std::string& str1, const std::string& str2);
+  static bool EqualsNoCase(const std::string& str1, const char* s2);
+  static bool EqualsNoCase(const char* s1, const char* s2);
```

Comment:
Why do you only fix these formatting issues, not all/none still contained in this header? Change seems completely random.